CHANGES.md

```diff
@@ -1,276 +1,2504 @@
+<ul>
+	<li><a href='https://codice.atlassian.net/browse/DDF-1802'>DDF-1802</a> - Metacards with GeometryCollection cause a cometd listener exception when rendered on both 2D and 3D maps
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2192'>DDF-2192</a> - Gazetteer service cannot handle large WKT
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2431'>DDF-2431</a> - Update/Delete will not finish if a source does not respond in a timely manner
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2519'>DDF-2519</a> - Fix NumberFormatException when a non-required metatype field is blank
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2563'>DDF-2563</a> - Path lengths in the distribution cause it to unzip improperly using windows&#39; built-in archive tool, forcing administrators to use a third-party tool
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2643'>DDF-2643</a> - TikaInputTransformer ingest fails on large files
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2650'>DDF-2650</a> - An invalid character in the URI could cause a content action to fail
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2778'>DDF-2778</a> - WMTS Imagery Providers do not project EPSG:4326 properly for Cesium
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2786'>DDF-2786</a> - Depending on the zoom level, system header and footer are sometimes cut off in the Catalog UI
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2788'>DDF-2788</a> - An improperly constructed GET request for a WSDL can cause an exception in the PEPAuthorizingInterceptor and keep it from logging the attempt
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2789'>DDF-2789</a> - The source poller can stop working when a source is removed because of a race condition with blueprint which causes sources to stop updating with new status
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2790'>DDF-2790</a> - Large Usernames in the Data Usage Admin App push table columns off the screen, causing erroneous behavior
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2797'>DDF-2797</a> - HTTP Proxy destroy method causes an exception when stopping endpoints which causes the Camel context to not shut down correctly
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2809'>DDF-2809</a> - URLs with extremely long paths can cause an exception in the web context policy manager
+	</li>
+	<li><a href='https://codice.atlassian.net/browse/DDF-2835'>DDF-2835</a> - CSW queries using extended attributes return 0 results, forcing integrators to use a different endpoint for advanced searches
+	</li>
+</ul>
+
+<h3>Story</h3>
+<ul>
+	<li><a href='https://codice.atlassian.net/browse/DDF-1765'>DDF-1765</a> - As a user, I want to add a layer to the Search UI map from a KML file so that I can visualize my KML data
+	</li>
+</ul>
+    
+<h3>New Feature</h3>
```

Comment:
It's just pulling from the types on the tickets, and I haven't edited any issues. (yet)